src/css/color-picker.less

```diff
@@ -150,3 +150,10 @@
   position: relative;
   z-index: 1;
 }
+
+// Eyeballed to be the same height as color preview and width when there are 4
+// colors per row.
+.color-picker-hex-input.form-control.input-sm {
+  width: 104px;
+  height: 28px;
+}
```

Comment:
do you really need all of those classes to select this element? did .color-picker-hex-input not work?